### DISCUSSION TITLE
First beta release

### DISCUSSION THREAD

**Discussion by paljsingh** [paljsingh/impartus-downloader#75] [General]
UI changes
- Pyside2/Qt5 based UI.
- Faster, responsive, better looking buttons.
- Round progressbar
- fontawesome based icons
- Supports system themes, works with dark and light modes.

Features
- Offline work to access previously download content without having to log in.
- Content Search.
- Option to view all attached slides.
- Option to download individual chat/captions files.
- More screen space, better organisation of the content.

<hr /><em>This discussion was created from the release <a href='https://github.com/paljsingh/impartus-downloader/releases/tag/v4.0-beta.1'>First beta release</a>.</em>

**Comment by unalivepool** [comment]
Thank you mate. This definitely augments learning. Thank you gain for your efforts. /\
